docs/development/pr-review.asciidoc

```diff
@@ -123,6 +123,7 @@ Establishing a comprehensive checklist for all of the things that should happen
 * Has thorough unit test coverage
 * Automated tests provide high confidence the change continues to work without manual verification
 * Appropriate product documentation is included (asciidocs)
+* If there's any doubts on the usability, performance, cross-browser compatibility of the change, pull the PR and test it
```

Comment:
All of these bullets are written about the PR itself rather than about what the reviewer should do. Can you rewrite this with that same intention?  Perhaps:

`Manual testing of the functionality in the event that there are any concerns about usability, performance, cross-browser issues, etc. has shown no problems`